start.sh

```diff
@@ -0,0 +1,9 @@
+DIR="data"
+
+if [ -d "$DIR" ]; then
+    rm -rf $DIR
+fi
+
+mkdir $DIR
+
+docker run --rm --name=scraper -v $PWD/data:/app/scraper/data liquidinvestigations/crji-avere-europarl python3 /app/scraper/main.py --id 124831
```

Comment:
Also, please add `--user="$(id -u):$(id -g)"` to the `docker run` arguments so the resulting files have your permission bits, otherwise they belong to `root`